packages/valory/skills/twitter_scoring_abci/models.py

```diff
@@ -60,6 +97,15 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.tweet_evaluation_round_timeout = self._ensure(
             "tweet_evaluation_round_timeout", kwargs, float
         )
+        self.openai_calls = OpenAICalls(
+            openai_call_window_size=self._ensure(
+                "openai_call_window_size", kwargs, float
+            ),
+            openai_calls_allowed_in_window=self._ensure(
+                "openai_calls_allowed_in_window", kwargs, int
+            ),
+        )
```

Comment:
Not a common pattern. Parameters are meant to be frozen, you could create the `OpenAICalls` object in the shared state using two params: `openai_call_window_size` and `openai_calls_allowed_in_window`